content/en/docs/components/model-registry/installation.md

```diff
@@ -18,28 +18,39 @@ These are the minimal requirements to install Model Registry:
 
 ## Installing Model Registry
 
-You can skip this step if you have already installed Kubeflow >=1.9. Your Kubeflow
-deployment includes Model Registry ([see tracker issue](https://github.com/kubeflow/manifests/issues/2631)).
+Kubeflow Model registry may be installed as part of a Kubeflow Platform, or as a standalone component.
+The best option for you will depend on your specific requirements.
 
-To install Model Registry as part of Kubeflow, follow the
-[Kubeflow installation guide](/docs/started/installing-kubeflow/).
+### Installing on Kubeflow Platform
+
+Kubeflow Model Registry is available as an opt-in alpha component in Kubeflow Platform 1.9+, see [Installing Kubeflow](/docs/started/installing-kubeflow/) to learn more about deploying the Kubeflow Platform.
+
+If you have deployed the Kubeflow manifests, you may follow [these instructions](https://github.com/kubeflow/manifests/tree/master/apps/model-registry/upstream#readme) to deploy Model Registry; please raise any feedback on [`kubeflow/model-registry`](https://github.com/kubeflow/model-registry/issues/new/choose).
+
+### Standalone installation
 
 If you want to install Model Registry separately from Kubeflow, or to get a later version
-of Model Registry, you can use one of the following Model Registry manifests.
-Remember to substitute the relevant release (e.g. `v0.1.2`), modify `ref=main` to `ref=v0.1.2`.
+of Model Registry, you can install the Model Registry manifests directly from the [Model Registry repository](https://github.com/kubeflow/model-registry).
 
-The following steps show how to install Model Registry in the context of a default Kubeflow >=1.8 installation.
+The manifests by defaut deploy the Model Registry in the `kubeflow` namespace;
+you must ensure the `kubeflow` namespace is available (e.g.: `kubectl create namespace kubeflow`)
```

Comment:
```suggestion
you must ensure the `kubeflow` namespace is available (for example: `kubectl create namespace kubeflow`)
```

Reply:
out of curiosity, in https://github.com/kubeflow/website/pull/3740#discussion_r1628170787 you suggested yourself using `e.g.:`
while this recommendation is replacing it with `for example:` 

... any particular guideline you are applying? 🤔